### ISSUE TITLE
[WCAG2.1]: Consider adding a performance budget or perf budget guidance to our testing criteria

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21181]
## Issue Description
With our new emphasis on mobile-first and WCAG2.1, I'd like to consider adding performance budget recommendations to app work.

Some examples:
* https://csswizardry.com/2020/01/performance-budgets-pragmatically/
* http://v3.danielmall.com/articles/how-to-make-a-performance-budget/

## Tasks
- [ ] Review linked examples
- [ ] Close ticket as `wont fix` OR
- [ ] Decide if performance budget is a good inclusion (binding? non-binding?)
- [ ] Document performance recommendations on VSP accessibility docs

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by noahgelman** [comment]
After reviewing the current state of performance on VA.gov I've concluded that we do not need to add a performance budget to our testing criteria.

VA.gov is above average in loading speed when compared to other websites. The content on VA.gov is mostly text content based which is not performant intensive. I cannot find any metric that would require us to add performance checks to our testing criteria.

That is not to say there is nothing that can be done to increase performance on the site. There are static assets that could be combined, minified, and cached. Adding height/width to images. Etc. But these are global issues that could be fixed separately to testing criteria and their actual negative impact on the website is minor.

Honestly, VA.gov is the most performant website I've worked on.
